src/Microsoft.Data.SqlClient/src/Microsoft.Data.SqlClient.csproj

```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+		<OutputType>ClassLibrary</OutputType>
+		<TargetFrameworks>net6.0;net8.0;net462</TargetFrameworks>
```

Comment:
I'm pretty sure there are netfx/netcore specific files - do you intend to disable default compile items and include specific files accordingly?

Reply:
Since the project isn't being built, it doesn't cause much issue to have files included.

If/when we eventually want to make this project the single build project, we will want to turn files on/off based on OS as well as netfx vs netcore. To me, conditional inclusion of the files in the project is not ideal since it makes it difficult to know at-a-glance what files are in the project. As such, I'd suggest using `#if` to wrap the files to include based on .net (already built-in), or OS (will require some consideration).